package.json

```diff
@@ -8,14 +8,15 @@
     "url": "http://github.com/codeclimate/codeclimate-eslint.git"
   },
   "dependencies": {
-    "babel-eslint": "^6.1.2",
+    "babel-eslint": "^7.0.0",
     "eslint": "3.11.1",
     "eslint-config-airbnb": "^11.0.0",
     "eslint-config-airbnb-base": "^7.0.0",
     "eslint-config-angular": "^0.5.0",
     "eslint-config-ember": "^0.3.0",
     "eslint-config-google": "^0.7.1",
     "eslint-config-hapi": "^10.0.0",
+    "eslint-config-react-app": "^0.5.0",
```

Comment:
@maxjacobson is this what you mean by adding an explicit dependency? Or does this dependency need to be defined elsewhere too?